### ISSUE TITLE
training memory

### ISSUE THREAD

**Issue by kwea123** [alibaba/cascade-stereo#8]
The numbers in the paper are all for testing with `batch size = 1`, right? It means it doesn't have gradient, and many operations are done inplace to save memory.
Do you remember the memory requirement for training with `batch size=2` and other default settings? I know I can download the repo and do it myself to see, but I think it's faster to ask you directly... If you remember the number it helps me a lot! Thank you.

**Comment by gxd1994** [comment]
hi @kwea123 , 
pytorch>=1.2 , 10G~11G when  batch size=2 in training stage.
